src/main/java/com/microsoft/sqlserver/jdbc/Geography.java

```diff
@@ -198,7 +211,7 @@ public boolean hasZ() {
     }
 
     /**
-     * Returns the X coordinate value.
+     * Gets the X coordinate value.
```

Comment:
I am sorry I made you change.....we're now going with "Returns the something of this thing." as per new guideline. You can leave it if you want I'm changing all the other files to be consistent I can change this after you merge.